src/dimensions/correlationdim.jl

```diff
@@ -189,16 +189,16 @@ end
 ################################################################################
 # Correlationsum, but we distributed data to boxes beforehand
 ################################################################################
-function boxed_correlationdim(data; M = size(data, 2), q = 2)
+function boxed_correlationdim(data; M = size(data, 2), q = 2, w = 0)
     r0 = estimate_r0_buenoorovio(data, M)
     ε0 = minimum_pairwise_distance(data)[1]
     εs = 10 .^ range(log10(ε0), log10(r0), length = 16)
-    boxed_correlationdim(data, εs, r0; M = M, q = q)
+    boxed_correlationdim(data, εs, r0; M = M, q = q, w = w)
```

Comment:
```suggestion
    boxed_correlationdim(data, εs, r0; M, q, w)
```

Julia 1.5+ has this nice feature that if the variable name and keyword name coincide, you don't have to type both.